sdgym/benchmark.py

```diff
@@ -504,6 +507,14 @@ def benchmark_single_table(synthesizers=DEFAULT_SYNTHESIZERS, custom_synthesizer
         pandas.DataFrame:
             A table containing one row per synthesizer + dataset + metric.
     """
+    if run_on_ec2:
+        if output_filepath is not None:
+            script_content = _create_sdgym_script(dict(locals()), output_filepath)
+            _create_instance_on_ec2(script_content)
```

Comment:
can we add the following unit tests:
1. Make sure it only runs this when the flag is set
2. Test the value error below
3. test the output of the `_create_sdgym_script`